src/components/TaskEditor.vue

```diff
@@ -39,7 +39,7 @@
         <TagSelector :task="taskEditorSectionIdAndTask.task" />
       </div>
       <template
-        v-for="(field, index) in taskEditorSectionIdAndTask.task.custom_fields"
+        v-for="(field, index) in project?.custom_fields"
```

Comment:
Because you changed to not allow undefined, all of these nullable ops should be able to be removed